Broken tests on i686-linux

`test_compressed_complex` is failing on i686-linux (python 3.10.9, pymemcache 4.0.0)
```
<...............................................................................>
pymemcache/test/test_serde.py::test_compressed_complex[serde0] FAILED
pymemcache/test/test_serde.py::test_compressed_complex[serde1] FAILED
pymemcache/test/test_utils.py::test_get_set PASSED
pymemcache/test/test_utils.py::test_get_set_unicide_key PASSED
pymemcache/test/test_utils.py::test_get_set_non_ascii_value PASSED
pymemcache/test/test_utils.py::test_get_many_set_many PASSED
pymemcache/test/test_utils.py::test_get_many_set_many_non_ascii_values PASSED
pymemcache/test/test_utils.py::test_add PASSED
pymemcache/test/test_utils.py::test_delete PASSED
pymemcache/test/test_utils.py::test_incr_decr PASSED
pymemcache/test/test_utils.py::test_prepand_append PASSED

=================================== FAILURES ===================================
_______________________ test_compressed_complex[serde0] ________________________
pymemcache/test/test_serde.py:147: in test_compressed_complex
    check(serde, CustomInt(sys.maxsize), FLAG_PICKLE | FLAG_COMPRESSED)
pymemcache/test/test_serde.py:32: in check
    assert flags == expected_flags
E   assert 1 == 9
_______________________ test_compressed_complex[serde1] ________________________
pymemcache/test/test_serde.py:147: in test_compressed_complex
    check(serde, CustomInt(sys.maxsize), FLAG_PICKLE | FLAG_COMPRESSED)
pymemcache/test/test_serde.py:32: in check
    assert flags == expected_flags
E   assert 1 == 9
=========================== short test summary info ============================
FAILED pymemcache/test/test_serde.py::test_compressed_complex[serde0] - assert 1 == 9
FAILED pymemcache/test/test_serde.py::test_compressed_complex[serde1] - assert 1 == 9
================ 2 failed, 460 passed, 156 deselected in 5.40s =================
```

```
>>> import sys
>>> sys.maxsize
2147483647
```

--- a/README.rst
+++ b/README.rst
@@ -137,6 +137,7 @@ Credits
 * `Hervé Beraud <https://github.com/4383>`_
 * `Martin Jørgensen <https://github.com/martinnj>`_
 * `Matej Spiller Muys <https://github.com/matejsp>`_
+* `misuzu <https://github.com/misuzu>`_
 
 We're Hiring!
 =============

--- a/pymemcache/test/test_serde.py
+++ b/pymemcache/test/test_serde.py
@@ -12,7 +12,6 @@ from pymemcache.serde import (
 )
 import pytest
 import pickle
-import sys
 import zlib
 
 
@@ -128,7 +127,7 @@ def test_compressed_complex(serde):
     check(serde, "£ $ €" * 10, FLAG_TEXT | FLAG_COMPRESSED)
 
     # test_int, doesn't make sense to compress
-    check(serde, sys.maxsize, FLAG_INTEGER)
+    check(serde, 9223372036854775807, FLAG_INTEGER)
 
     # test_pickleable
     check(
@@ -144,4 +143,4 @@ def test_compressed_complex(serde):
 
     # test_subtype
     # Subclass of a native type will be restored as the same type
-    check(serde, CustomInt(sys.maxsize), FLAG_PICKLE | FLAG_COMPRESSED)
+    check(serde, CustomInt(9223372036854775807), FLAG_PICKLE | FLAG_COMPRESSED)

